standalone/src/models/custom-application/custom-application/index.ts

```diff
@@ -1,5 +1,7 @@
-export * as CustomApplicationDraft from './custom-application-draft';
+import { GraphqlModelBuilder } from './builders';
+import * as CustomApplicationPresets from './presets';
 
-export { default as random } from './builder';
-export { default as presets } from './presets';
-export * from './types';
+export const CustomApplicationGraphql = {
```

Comment:
I think I missed some information when you asked me about having only one model named with the GraphQL suffix.
Since the package was already exporting the model as `CustomApplication`, that's something we can't remove or we will break all consumers.

In this case I think we need to keep the new one but have an alias for it so we don't break consumers.

We can have a chat about it to make sure we understand what we need to do.

Reply:
fixed in b02bf8b02d41a325a35f359de5555c514328a6cb (updated all exports accordingly)